### PR TITLE
Persons in bdlaethc14

### PR DESCRIPTION
**Pull request by MassimoVilla**
Four individuals mentioned in ms BDLaeth c14

### CODE REVIEW

File: new/PRS13182WaldaSe.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ ሥላሴ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Śǝllāse</persName>
+                    <persName><roleName type="title">ʾabbā</roleName></persName>
+                    <faith type="EOTC"/>      
+                    <floruit notBefore="1800" notAfter="1970"/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13182WaldaSe" passive="BDLaethc14#a2">
+                        <desc>The name of ʾabbā Walda Śǝllāse is mentioned in an additional note, written on the margin in a recent hand.</desc></relation>
```
Comment (SusanneHummel) [inline new/PRS13182WaldaSe.xml:56]: According to the new Aethiopica-guidelines (and thus to the HLCEES rules), we write the tites in capital, i.e. : ʾAbbā.

I would suggest to follow this rule, but the last word is with our native speaker @Gnisci

File: new/PRS13183Tasamma.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተሰማ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Tasammā</persName>
+                    <persName><roleName type="title">ʾalaqā</roleName></persName>
+                    <faith type="EOTC"/>      
+                    <floruit notBefore="1800" notAfter="1970"/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13183Tasamma" passive="BDLaethc14#a2">
+                        <desc>The name of ʾalaqā Tasammā is mentioned in an additional note, written on the margin in a recent hand.</desc></relation>
```
Comment (SusanneHummel) [inline new/PRS13183Tasamma.xml:56]: same comment to the title as above

**Review by SusanneHummel — APPROVED** [review]
Thank you!

File: new/PRS13182WaldaSe.xml (hunk 1 above)
Comment (thea-m) [reply new/PRS13182WaldaSe.xml:56]: I'm happy to follow @Gnisci 's decision on what reads best.

**Review by ElianaDalSasso — APPROVED** [review]

**Comment by MassimoVilla** [comment]
Merged. Thank  you for your comments. I have taken the freedom to add a new point to our meeting agenda: Titles written in in capital letter (i.e. : ʾAbbā)? So, as soon as we reach a shared decision, I can go through all titles in the old files and open a new PR for that.

**Pull request merged**